project/api/models/representative.py

```diff
@@ -50,11 +48,36 @@ class Representative(models.Model):
     def __str__(self):
         return "Rep - {}".format(self.name)
 
-    def summarize(self):
-        return {
+    def summarize(self, account=None):
+        from .vote import Vote
+
+        data = {
             "bioguide_id": self.bioguide_id,
             "name": self.name,
             "party": self.party,
             "state": '{state}'.format(state=dict(US_STATES).get(self.state, self.state)),
             "profile_image": "https://theunitedstates.io/images/congress/225x275/{}.jpg".format(self.bioguide_id)
         }
+        if account:
+            votes = {}
+            bills = account.get_voted_bills(serialize=False)
+            voted_same_count = Vote.objects.filter(
+                bill__in=bills["supported_bills"], representative=self, vote="yes"
+            ).count()
+            voted_same_count += Vote.objects.filter(
+                bill__in=bills["opposed_bills"], representative=self, vote="no"
+            ).count()
+
+            voted_different_count = Vote.objects.filter(
+                bill__in=bills["supported_bills"], representative=self, vote="no"
+            ).count()
+            voted_different_count += Vote.objects.filter(
+                bill__in=bills["opposed_bills"], representative=self, vote="yes"
+            ).count()
+
+            votes["voted_same"] = voted_same_count
+            votes["voted_different"] = voted_different_count
```

Comment:
Can we add a `votes["compatibility_score"] = voted_same_count / (voted_same_count + voted_different_count)`? I think this would be a good "summary statistic" to show the user.